commons-rng-core/src/main/java/org/apache/commons/rng/core/source32/PcgXshRr32.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.rng.core.source32;
+
+/**
+ * XSH-RR: An xorshift mixes some high-order bits down,
```

Comment:
This class (and the other public generators) could do with an introduction line since the introduction line in the AbstractPCG will not be part of the public Javadoc. The actual transform details are not complete (what is the shift to mix some bits down) and so should be removed. A user can read the link you have provided to find out more, or read the source code. I would adapt the summary line from the abstract class here and explain what PCG means, e.g
```
A Permutated Congruential Generator (PCG) that uses the XSH-RR output transformation to create 32-bit output.

This is a member of the PCG suite of generators, a family of simple fast space-efficient statistically good algorithms for random number generation.
```

Add a blank line before the `@see` tag.

Reply:
Done. Kindly check.